.changeset/tricky-months-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@microsoft/atlas-css": patch
```

Comment:
This is a minor update, because we're adding functionality.

Reply:
Fixed!